crates/uv-requirements/src/lookahead.rs

```diff
@@ -100,21 +107,28 @@ impl<'a, Context: BuildContext + Send + Sync> LookaheadResolver<'a, Context> {
         let mut seen = FxHashSet::default();
 
         // Queue up the initial requirements.
-        let mut queue: VecDeque<Requirement> = self
+        let mut queue: VecDeque<_> = self
             .constraints
             .apply(self.overrides.apply(self.requirements))
             .filter(|requirement| requirement.evaluate_markers(markers, &[]))
-            .chain(self.editables.iter().flat_map(|(editable, metadata)| {
-                self.constraints
-                    .apply(self.overrides.apply(&metadata.requires_dist))
-                    .filter(|requirement| requirement.evaluate_markers(markers, &editable.extras))
-            }))
+            .chain(
+                self.editables
+                    .iter()
+                    .flat_map(|(editable, _metadata, requirements)| {
+                        self.constraints
+                            .apply(self.overrides.apply(&requirements.dependencies))
+                            .filter(|requirement| {
+                                requirement.evaluate_markers(markers, &editable.extras)
+                            })
+                    }),
+            )
             .cloned()
             .collect();
 
         while !queue.is_empty() || !futures.is_empty() {
             while let Some(requirement) = queue.pop_front() {
-                if matches!(requirement.version_or_url, Some(VersionOrUrl::Url(_))) {
+                // TODO(konsti): Git and path too
```

Comment:
This is blocking, no?

Reply:
Outdated command, the condition is already fixed